.github/workflows/README.md

```diff
@@ -0,0 +1,49 @@
+# CI/CD Pipeline
+
+## Overview
+
+The pipeline is defined in the `.github/workflows` directory. It is structured as follows:
+
+- all `0*.yml` files are the main pipeline definitions, that are triggered by either pushing code, creating a pull request, manually running a job, or a scheduled job.
+- all `1*.yml` files are part of a reusable pipeline definition, that is included in the main pipeline definitions.
+- all `2*.yml` files are part of miscellaneous job definitions, also included in the main pipeline definitions.
+
+Additionally there are custom actions defined in the `.github/actions` directory, that are used in the pipeline definitions.
+
+## Pipeline Definitions
+
+The main pipeline definitions are as follows:
+
+- `01_pr.yaml`: Triggered by creating a pull request against master and develop. It runs the reusable CI pipeline.
+- `02_push.yaml`: Triggered by pushing code to master or develop and is otherwise identical to the `00_pr.yaml` pipeline.
+- `03_release.yaml:`: Triggered by pushing a tag prefixed with `v` and builds the Connaisseur image, checks that the image tag matches the git tag, runs the integration tests, publishes the helm chart to ArtifactHub and updates the Connaisseur documentation.
+- `04_publish.yaml`: Triggered manually and publishes the helm chart to ArtifactHub and updates the Connaisseur documentation. This is meant to be used in case the `03_release.yaml` pipeline fails and the helm chart needs to be published manually.
+- `05_nightly.yaml`: Triggered by a scheduled job that runs every night at 01:30, which runs the reusable CI pipeline but without unit tests, sast jobs, documentation publishing and integration tests. This pipeline also cleans up old images from the GitHub container registry.
+- `06_dockerhub_check.yaml`: Triggered by a scheduled job that runs every wednesday at 06:37, which pulls all test images from DockerHub to make sure they are still available and don't get deleted because of inactivity.
```

Comment:
```suggestion
- `06_dockerhub_check.yaml`: Triggered by a scheduled job that runs weekly, which pulls all test images from DockerHub to make sure they are still available and don't get deleted because of inactivity.
```